waspc/src/Wasp/Error.hs

```diff
@@ -0,0 +1,66 @@
+module Wasp.Error (showErrorForTerminal) where
+
+import Data.List (intercalate)
+import StrongPath (Abs, File', Path')
+import qualified StrongPath as SP
+import Wasp.Analyzer.Parser.Ctx (Ctx, getCtxRgn)
+import Wasp.Analyzer.Parser.SourcePosition (SourcePosition (..))
+import Wasp.Analyzer.Parser.SourceRegion (SourceRegion (..))
+import Wasp.Util (indent, insertAt, leftPad)
+import qualified Wasp.Util.Terminal as T
+
+showErrorForTerminal :: (Path' Abs File', String) -> (String, Ctx) -> String
```

Comment:
Here I implement function which prints a Wasp error in nice fashion, similar to how haskell errors are printed, with file name, location, error message, and also snippet of code where error happened with error clearly marked in red (and line numbers).

Reply:
I should add comments for this function.